app/services/google/scrape_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Google
+  class ScrapeService
+    attr_reader :keyword
```

Comment:
Is it keyword or keyword_id 🤔 

```suggestion
    attr_reader :keyword_id
```

Reply:
Fixed in latest version